files/build_templates/docker_image_ctl.j2

```diff
@@ -201,8 +201,19 @@ function postStartAction()
            ip link set dev ns-eth1"$NET_NS" netns "$NET_NS"
            ip netns exec "$NET_NS" ip link set ns-eth1"$NET_NS" name eth1
 
+           if [[ -n "$lc_ip_offset" ]]; then
```

Comment:
@anamehra could you point where this variable lc_ip_offset is populated ?

Reply:
The offset needs to be set in platform/chassisdb.conf
Updated the description.